stubs/PDOStatement.stub

```diff
@@ -7,5 +7,16 @@
  */
 class PDOStatement implements Traversable, IteratorAggregate
```

Comment:
would you agree adding a class-level generic for PDOStatement?

```
/**
 * @template rowType
 *
 * @implements Traversable<int, rowType>
 * @implements IteratorAggregate<int, rowType>
 *
 * @link https://php.net/manual/en/class.pdostatement.php
 */
 ```

I need the class to be generic for phpstan-dba to enrich this type with real-time db schema information.

Reply:
I have no idea how this class is used in practice in real-world code so I can't decide this :/